wallets/ethereum-wallet-mock/test/e2e/switchNetwork.spec.ts

```diff
@@ -0,0 +1,34 @@
+import { ANVIL_CHAIN_ID, ANVIL_URL_URL } from '../../src'
+
+import test from '../synpress'
+
+const { expect } = test
+
+function createAnvilNetwork() {
+  return {
+    name: 'Anvil',
+    rpcUrl: ANVIL_URL_URL,
+    chainId: ANVIL_CHAIN_ID,
+    blockExplorerUrl: 'https://etherscan.io/',
+    nativeCurrency: {
+      decimals: 18,
+      name: 'Anvil',
+      symbol: 'ETH'
+    }
+  }
+}
+
+test('should switch network', async ({ walletMock, page }) => {
+  const network = createAnvilNetwork()
+
+  await walletMock.addNetwork(network)
+
+  await walletMock.switchNetwork(network.name)
+
+  const chainId = await page.evaluate(async () => {
+    return await window.ethereum.request({ method: 'eth_chainId' })
+  })
+
+  // Mocked BSC chain id due to https://github.com/DePayFi/web3-mock/issues/33
+  expect(chainId).toBe('0x38')
```

Comment:
i hate this 🐕

Reply:
Yeah, me too. They did not answer if I will have to push a fix to their repo.

Ref: https://github.com/DePayFi/web3-mock/issues/33

Reply:
change to Optimism chain id